frontend/src/App.css

```diff
@@ -1,27 +1,3 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
```

Comment:
I see that this project has Bootstrap installed as its CSS library, where a lot of this custom CSS, especially dealing with grid/flex, is already available: https://getbootstrap.com/docs/4.4/layout/grid/.

Did we want to uninstall bootstrap and stick with custom CSS for this project?